Makefile.am

```
@@ -42,6 +48,4 @@ endif
 if CONFIG_LSB
 	chmod +x $(DESTDIR)$(lsbrcdir)/jacklistener
 endif
-if CONFIG_SYSTEMD
-	chmod +x $(DESTDIR)$(sbindir)/jacklistener-runscript
-endif
+chmod +x $(DESTDIR)$(sbindir)/jacklistener-runscript
```

make requires a tab here, otherwise I get the following error at this line:

```
Makefile:1117: *** missing separator.  Stop.
```

```suggestion
	chmod +x $(DESTDIR)$(sbindir)/jacklistener-runscript
```